src/components/animate/AnimatePresence.vue

```diff
@@ -75,13 +75,18 @@ export default defineComponent({
       // on mount we set initial values, but the issue is that enter will run at
       // the same time, setTimeout(0) makes the animation run on the next
       // available tick, so it's instant visually but on a tick delay for code
+      emit('on-presence', { isCompleted: false });
```

Comment:
What is this `isCompleted`? It doesn't seem to be used in the functions called from this emit.